website/content/en/preview/testing-guide

```diff
@@ -0,0 +1,41 @@
+# Karpenter Testing User Guide
+
+Currently, users can only test Karpenter by adding to the list of integration tests run in a mock environment or installing and testing Karpenter on a real cluster. Users that want to test more are limited by the lack of automation and configurability, resulting in much less testing.
```

Comment:
This "Users that want to test more ..., resulting in much less testing" sounds a bit redundant. May want to rephrase it. For example.

"Users who want to run more comprehensive tests are limited by the lack of test automation and configurability."